presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeHiveExternalTableTpchQueries.java

```diff
@@ -114,7 +113,9 @@ public void tearDown()
         for (String tableName : TPCH_TABLES) {
             dropTableIfExists(javaQueryRunner, HIVE, TPCH, tableName);
         }
-        assertUpdate(format("DROP SCHEMA IF EXISTS %s.%s", HIVE, TPCH));
+
+        // https://github.com/prestodb/presto/issues/23908
```

Comment:
Remove before land? Thanks!

Reply:
I commented the following line to unblock. Created an issue to resolve it properly: https://github.com/prestodb/presto/issues/23908

Let me leave a comment on the issue to remove this comment once resolved.

Reply:
https://github.com/prestodb/presto/issues/23908#issuecomment-2444804185